modules/install-sno-installing-with-usb-media.adoc

```diff
@@ -4,34 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="installing-with-usb-media_{context}"]
-= Installing with USB media
+= Creating a bootable ISO image on a USB drive
 
-Installing with USB media involves creating a bootable USB drive with the discovery ISO on the administration node. Booting the server with the USB drive prepares the node for a single node installation.
+You can install the {sno} cluster using a bootable USB drive that contains the ISO image. Booting the server with the USB drive prepares the node for a single node installation.
```

Comment:
I think we should remove the word OpenShift from this paragraph and maybe change the wording a little bit to clarify that it's just generic, non-openshift related section on how to boot ISO files from USB drives